marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-vis/graph-vis.tsx

```diff
@@ -571,25 +571,21 @@ const GraphVis: React.FC<Props> = (props) => {
           setCoords(newCoords);
           props.saveEntityCoords(nodes[0], positions.x, positions.y);
         }
+      } else {
+        if (props.entityTypes) {
+          let newCoords = {};
```

Comment:
@AkshayBajajML, in the pan case we aren't updating the coords state with setCoords, I think we can remove the newCoords init here and also line 579 below?

Reply:
Done